core/src/main/resources/reference.conf

```diff
@@ -253,6 +253,25 @@ akka.persistence.r2dbc {
     # Enabling this has some performance overhead.
     # A fast query for Postgres is "SELECT 1"
     validation-query = ""
+
+
+    # Abort any statement that takes more than the specified amount of time.
+    # This timeout is handled by the database server.
+    # This timeout should be less than `close-calls-exceeding`.
+    statement-timeout = off
```

Comment:
I didn't enable this by default, but I would guess that it's good to have, and should probably be slightly less then the circuit breaker timeout (default 10 seconds)